frontend/packages/dev-console/src/components/buildconfig/BuildConfigFormEditor.tsx

```diff
@@ -13,19 +13,40 @@ import SecretsSection from './sections/SecretsSection';
 import SourceSection from './sections/SourceSection';
 import TriggersSection from './sections/TriggersSection';
 
-export type BuildConfigFormEditorProps = {
+type BuildConfigFormEditorProps = {
   namespace: string;
 };
 
-const BuildConfigFormEditor: React.FC<BuildConfigFormEditorProps> = ({ namespace }) => {
+const List: React.FC<{ namespace: string }> = ({ namespace }) => {
```

Comment:
In both other components you reused the parent component props. Not required, but for consistency you can do this here as well

```suggestion
const List: React.FC<BuildConfigFormEditorProps> = ({ namespace }) => {
```